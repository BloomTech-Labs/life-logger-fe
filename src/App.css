.App {
  display: flex;
  margin: 0 auto;
  justify-content: center;
  text-align: center;
}


.dark-mode .dark-mode__toggle {
  background: #292929 !important;
}

.dark-mode .toggle {
  background-image: linear-gradient(315deg, #1F2022 0%, white 250%)!important;
}

.toggled {
  left: 18px;
}

.dark-mode {
  color: #fff!important;
  background: #292929 !important;

}

.dark-mode .navbar {
  background-color: #1F2022 ;
  background-image: linear-gradient(315deg, #1F2022 0%, #1F2022 74%);
  border: none;
}

.dark-mode .task-title-header {
  border-bottom: 1.5px solid #1F2022 ;
}

.dark-mode input {
  background-color:  #1F2022;
  color: white;
  border: none !important;
}

.dark-mode textarea {
  background-color:  #1F2022;
  color: white;
  border: none;
}

.dark-mode select {
  background-color:  #1F2022;
  color: white;
  border: none;
}

.dark-mode .fc-event, .fc-event-dot{
  background-image: linear-gradient(315deg, #045de9 0%, #09c6f9 74%);
  border: 1px solid #09c6f9;
}

.dark-mode .fc-unthemed td.fc-today{
  background-color: rgb(0, 0, 0);
}