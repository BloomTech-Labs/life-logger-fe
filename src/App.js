/** @jsx jsx */
import { jsx } from 'theme-ui';
import { Fragment } from 'react';
import { Route, Switch } from 'react-router-dom';

import SignUpForm from './components/forms/SignupForm';
import PrivateRoute from './PrivateRoute';
import Dashboard from './components/dashboard/Dashboard';
import LandingPage from './components/LandingPage';
import LoginForm from './components/forms/LoginForm';
import CreateTask from './components/forms/CreateTask';
import AboutUs from './AboutUs';
import Header from './components/header/Header';

const App = () => {
  return (
    <Fragment>
      <Header />
      <Switch>
        <Route exact path="/" component={LandingPage} />
        <Route path="/aboutus" component={AboutUs} />
        <PrivateRoute exact path="/dashboard" component={Dashboard} />
        <Route path="/signup" component={SignUpForm} />
        <Route path="/login" component={LoginForm} />
        <PrivateRoute path="/createtask" component={CreateTask} />
      </Switch>
    </Fragment>
  );
};
export default App;
