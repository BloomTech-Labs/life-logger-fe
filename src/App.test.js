import { renderWithRouter } from '../tests/routerTestsUtil';
import App from './App';

describe('App component tests', () => {
  it('renders without crashing', () => {
    renderWithRouter(App, {
      route: '/',
      path: '/',
    });
  });
});
