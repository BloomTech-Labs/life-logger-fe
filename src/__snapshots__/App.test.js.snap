// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<styled.div
  className="app-container"
  id="container"
>
  <Header />
  <PrivateRoute
    component={[Function]}
    exact={true}
    path="/"
  />
  <PrivateRoute
    component={[Function]}
    path="/task/:id"
  />
  <PrivateRoute
    component={[Function]}
    path="/edit-task/:id"
  />
  <PrivateRoute
    component={[Function]}
    path="/calendar"
  />
  <PrivateRoute
    component={[Function]}
    path="/settings"
  />
</styled.div>
`;
